app/assets/stylesheets/companies/companies_index.scss

```diff
@@ -0,0 +1,424 @@
+
+        &:nth-child(1) {
+          margin-bottom: calc(10px + (20 - 10) * ((100vw - 900px) / (1822 - 900)));
+        }
+
+        &:nth-child(2) {
+          margin-left: calc(60px + (100 - 60) * ((100vw - 900px) / (1822 - 900)));
+          margin-bottom: calc(10px + (20 - 10) * ((100vw - 900px) / (1822 - 900)));
+        }
+
+        &:nth-child(4) {
+          margin-left: calc(60px + (100 - 60) * ((100vw - 900px) / (1822 - 900)));
+        }
+
+        .info-number {
+          display: flex;
+          align-items: center;
+          font-style: normal;
+          font-weight: 700;
+          font-size: calc(60px + (74 - 60) * ((100vw - 320px) / (1920 - 320)));
+          line-height: 87px;
+
+          &:hover {
+            color: $base-color;
+          }
+        }
+
+        .info-text {
+          display: flex;
+          align-items: center;
+          font-size: calc(11px + (15 - 11) * ((100vw - 320px) / (1920 - 320)));
+          line-height: 19px;
+          white-space: nowrap;
+
+          &:hover {
+            color: $base-color;
+          }
+        }
+      }
+    }
+
+    .container-about {
+      display: flex;
+      flex-wrap: nowrap;
+      align-items: center;
+      justify-content: center;
+      position: relative;
+      width: 760px;
+      height: 460px;
+      border: 10px solid $color-company;
+      cursor: auto;
+
+      .text-about {
+        display: flex;
+        align-items: center;
+        font-size: 23px;
+        line-height: 28px;
+        text-align: justify;
+
+        @media screen and (max-width: 1450px) {
+          font-size: calc(14px + (23 - 14) * ((100vw - 320px) / (1920 - 320)));
+        }
+      }
+    }
+  }
+
+  @media screen and (max-width: 1163px) {
+
+    .wrapper-company {
+
+      &:nth-child(odd) {
+
+        .container-about {
+          border-radius: 40px;
+        }
+      }
+
+      &:nth-child(even) {
+
+        .container-about {
+          border-radius: 40px;
+        }
+      }
+
+      .container-info {
+        height: auto;
+        width: auto;
+        min-width: 350px;
+      }
+
+      .container-about {
+        height: auto;
+        width: auto;
+      }
+    }
+  }
+
+  @media screen and (max-width: 899px) {
+
+    .wrapper-company {
+
+      &:nth-child(odd) {
+        flex-direction: column;
+
+        .container-about {
+          margin-left: 0;
+          border-radius: 0 0 40px 40px;
+          border-top: 0;
+          border-right: 10px solid $color-company;
+          border-bottom: 10px solid $color-company;
+          border-left: 10px solid $color-company;
+        }
+      }
+
+      &:nth-child(even) {
+        flex-direction: column;
+
+        .container-about {
+          border-top: 0;
+          border-right: 10px solid $color-company;
+          border-bottom: 10px solid $color-company;
+          border-left: 10px solid $color-company;
+          margin-right: 0;
+          border-radius: 0 0 40px 40px;
+        }
+      }
```

Comment:
too many duplicated styling rules

Reply:
Unfortunately, when I put these styles in the general selector, these rules did not work